### ISSUE TITLE
A few problems of the code

### ISSUE THREAD

**Issue by WYL-Projects** [LMissher/ST-GRAT#4]
Hello, author,
  As far as I can read, I find there may be the following problems in this code.
  Firstly,  I find that the ratio of training set / verification set / test set in the paper is 7 / 1 / 2, but the data distribution ratio in this code is 6 / 2 / 2.
  Secondly, whether the results in the paper are the best results measured from the validation data set or the last test set, or the best results selected from the validation set and test set!
  Finally, hope to get an answer!
  Thanks!
